test/e2e/framework/framework.go

```diff
@@ -259,6 +259,45 @@ func (f *Framework) matchNginxConditions(name string, matcher func(cfg string) b
 	}
 }
 
+// Get the ingress-nginx nginx-configuration configMap
+func (f *Framework) GetNginxConfigMap() (*v1.ConfigMap, error) {
+	config, err := f.KubeClientSet.CoreV1().ConfigMaps("ingress-nginx").Get("nginx-configuration", metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	if config == nil {
+		return nil, fmt.Errorf("nginx-configuration configMap does not exist")
+	}
+
+	return config, err
+}
+
+// UpdateNginxConfigMap updates the nginx-configuration's Data field
+func (f *Framework) UpdateNginxConfigMapData(k, v string) (error) {
```

Comment:
nitpick for consistancy: `(k string, v string)`

Reply:
sounds good :)